### PR TITLE
PAR-205: Added base fields for Advice, Inspection Plans and People.

### PR DESCRIPTION
**Pull request by kalpaitch**
This should cover all of the field types required in the model, so provide these work and look good it should be relatively straight forward for the rest.

### CODE REVIEW

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php [hunk 1]
```diff
@@ -63,4 +65,248 @@
...
+      ->setTranslatable(FALSE)
+      ->setDisplayOptions('form', [
+        'type' => 'boolean_checkbox',
+        'weight' => 6,
+      ]);
+
+    // Trading Name
+    $fields['trading_name'] = BaseFieldDefinition::create('string')
+      ->setLabel(t('Trading Name'))
+      ->setDescription(t('The tradig names for this Organisation.'))
```
Comment (stemount) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: the tradig names? and Name(s) (plural)?

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php [hunk 2]
```diff
@@ -63,4 +65,248 @@
...
+      ->setDefaultValue('')
+      ->setDisplayOptions('form', [
+        'type' => 'string_textfield',
+        'weight' => 7,
+      ])
+      ->setDisplayConfigurable('form', FALSE);
+
+    // Reference to Primary Person
+    $fields['primary_person'] = BaseFieldDefinition::create('entity_reference')
+      ->setLabel(t('Primary Contacts'))
```
Comment (stemount) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: key `primary_person` with Primary Contacts as a label and description is singular as well.

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php [hunk 3]
```diff
@@ -63,4 +65,248 @@
...
+          'match_operator' => 'CONTAINS',
+          'size' => 60,
+          'placeholder' => '',
+        ),
+      ));
+
+    // Reference to Person
+    $fields['person'] = BaseFieldDefinition::create('entity_reference')
+      ->setLabel(t('Contacts'))
+      ->setDescription(t('The secondary contacts for this Organisation.'))
```
Comment (stemount) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: contact(s)?

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php [hunk 4]
```diff
@@ -63,4 +65,248 @@
...
+        'settings' => array(
+          'match_operator' => 'CONTAINS',
+          'size' => 60,
+          'placeholder' => '',
+        ),
+      ));
+
+    // Reference to SIC Code
+    $fields['sic_code'] = BaseFieldDefinition::create('entity_reference')
+      ->setLabel(t('SIC Codes'))
```
Comment (stemount) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: plural?

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php (hunk 2 above)
Comment (kalpaitch) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: Yea will change this label I think to 'Primary Person'

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php (hunk 3 above)
Comment (kalpaitch) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: Yea will change to 'Alternative Contact' I think. Jamie mentioned we don't really want to call them secondary contacts.

File: web/modules/custom/par_data/src/Entity/ParDataOrganisation.php (hunk 4 above)
Comment (kalpaitch) [inline web/modules/custom/par_data/src/Entity/ParDataOrganisation.php]: Yup, will make singular, thanks.

**Pull request merged**